docusaurus/docs/React/components/core-components/channel.mdx

```diff
@@ -279,6 +287,15 @@ Custom UI component to be displayed when the `MessageList` or `VirtualizedMessag
 |-----------|---------------------------------------------------------------------------------------------------|
 | component | <GHComponentLink text='EmptyStateIndicator' path='/EmptyStateIndicator/EmptyStateIndicator.tsx'/> |
 
+### enrichURLForPreview
+
+A global flag to toggle the URL enrichment and link previews in `MessageInput`. By default, the feature is disabled. Can be overridden on Thread and MessageList level through `additionalMessageInputProps`
```

Comment:
```suggestion
A global flag to toggle the URL enrichment and link previews in `MessageInput`. The feature is disabled by default. It can be overridden on `Thread` and `MessageList` level through `additionalMessageInputProps`
```